### PR TITLE
MODSOURCE-67 Remove 'errorMessages' from response to get records

### PR DESCRIPTION
**Pull request by Anatolii-Starkov**
-added new schema recordBatch.json
-removed errorMessages field from recordCollection.json

### CODE REVIEW

File: schemas/dto/recordBatch.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Collection of record DTO",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "records": {
+      "description": "List of records",
```
Comment (Igor-Gorchakov) [inline schemas/dto/recordBatch.json]: List of successfully saved records

Comment (OleksiiKuzminov) [inline schemas/dto/recordBatch.json]: List of saved records

**Comment by OleksiiKuzminov** [comment]
add and fix examples plz

File: schemas/dto/recordBatch.json (hunk 1 above)
Comment (Anatolii-Starkov) [reply schemas/dto/recordBatch.json]: Do you suggest me to create a separate schema for update (if it's needed) and write there "List of updated records"? Schema per operation?

Comment (OleksiiKuzminov) [reply schemas/dto/recordBatch.json]: List of successfully processed records?

**Pull request merged**